components/FlashcardForm.js

```diff
@@ -0,0 +1,57 @@
+import { collections } from "@/lib/db/collections";
+import styled from "styled-components";
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 4px;
+`;
+
+export default function FlashcardForm({ onSubmit }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    onSubmit(data);
+  }
+
+  return (
+    <StyledForm onSubmit={handleSubmit}>
+      <label htmlFor="question">Question:</label>
+      <input
+        id="question"
+        type="text"
+        name="question"
+        placeholder="Question*"
+        required
+      ></input>
```

Comment:
inputs should be self closing
```suggestion
      <input
        id="question"
        type="text"
        name="question"
        placeholder="Question*"
        required
      />
```
Please change this for every input.

Reply:
Done